src/lib.rs

```diff
@@ -302,38 +302,85 @@ impl Database {
     /// It will simply skip malformed fonts and will print a warning into the log for each of them.
     #[cfg(feature = "fs")]
     pub fn load_fonts_dir<P: AsRef<std::path::Path>>(&mut self, dir: P) {
-        self.load_fonts_dir_impl(dir.as_ref())
+        self.load_fonts_dir_impl(dir.as_ref(), &mut Default::default())
+    }
+
+    #[cfg(feature = "fs")]
+    fn canonicalize(
+        &self,
+        path: std::path::PathBuf,
+        entry: std::fs::DirEntry,
+        seen: &mut std::collections::HashSet<std::path::PathBuf>,
+    ) -> Option<(std::path::PathBuf, std::fs::FileType)> {
+        let file_type = entry.file_type().ok()?;
+        if !file_type.is_symlink() {
+            if seen.capacity() != 0 {
```

Comment:
`if !seen.is_empty()`?

Reply:
I know, but it shouldn't matter in this case?

Reply:
It should be fine, changed.